asv_benchmarks/asv.conf.json

```diff
@@ -28,9 +28,9 @@
     //     "PIP_NO_BUILD_ISOLATION=false python -mpip wheel --no-deps --no-index -w {build_cache_dir} {build_dir}"
     // ],
 
-    // List of branches to benchmark. If not provided, defaults to "master"
+    // List of branches to benchmark. If not provided, defaults to "main"
```

Comment:
I think asv will still default to master :(
```suggestion
    // List of branches to benchmark. If not provided, defaults to "master
```
It should not matter since we always specify the branch we want to benchmark